lib/vdsm/storage/lvm.py

```diff
@@ -37,7 +37,6 @@
 import threading
 
 from itertools import chain
-import six
```

Comment:
This is a good change but it does not belong to this PR. Please send it as a separate PR.